.github/workflows/tests.yml

```diff
@@ -0,0 +1,26 @@
+name: tests
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+      - name: Go to test directory
+        run: cd stubway.tests
```

Comment:
stubway.tests contains integration tests and AEM instance (with stubway deployed) is needed to run them. So we cannot  run these tests. We should runn tests from `core` directory.

Reply:
Yep. I was experimenting with this. For now waiting for unit tests in core